azure-pipelines.yml

```
@@ -3,6 +3,68 @@
 
 name: K4A-SDK-$(SourceBranchName)-$(Date:yyyyMMdd)-$(Rev:rrr)
 
+schedules:
+- cron: "30 8 * * *"
```

I think we can simplify all this into one cron schedule

```suggestion
- cron: "*/30 08-13 * * *"
```

Check out this website for more info on cron expressions

https://crontab.guru/#*/30_08-13_*_*_*

---

BTW, thanks for the  tip, that cleaned things up a lot! I forgot to mention that before I submitted last night

---

Just trying to turn your into a Linux SysAdmin 😄